cmd/authctl/user/disable.go

```diff
@@ -0,0 +1,34 @@
+package user
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/spf13/cobra"
+	"github.com/ubuntu/authd/internal/consts"
+	"github.com/ubuntu/authd/internal/proto/authd"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials/insecure"
+)
+
+// DisableCmd is a command to disable a user.
+var DisableCmd = &cobra.Command{
+	Use:   "disable",
+	Short: "Disable a user managed by authd",
+	Args:  cobra.ExactArgs(1),
+	RunE: func(cmd *cobra.Command, args []string) error {
+		fmt.Printf("Disabling user %s\n", args[0])
```

Comment:
Maybe use `%q`? Otherwise we can just rely on `fmt.Println`

Reply:
Changed to %q. Thanks.